ChangeFeedLabSolution/ChangeFeedProcessor.cs

```diff
@@ -10,65 +10,110 @@
 /// </summary>
 namespace ChangeFeedFunction
 {
+    using System;
     using System.Collections.Generic;
-    using System.Configuration;
-    using System.Text;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Azure.Messaging.EventHubs;
+    using Azure.Messaging.EventHubs.Producer;
     using Microsoft.Azure.Documents;
-    using Microsoft.Azure.EventHubs;
     using Microsoft.Azure.WebJobs;
-    using Microsoft.Azure.WebJobs.Host;
-    using Newtonsoft.Json;
+    using Microsoft.Extensions.Logging;
 
     /// <summary>
     /// Processes events using Cosmos DB Change Feed.
     /// </summary>
-    public static class ChangeFeedProcessor
+    public class ChangeFeedProcessor
     {
         /// <summary>
         /// Name of the Event Hub.
         /// </summary>
-        private static readonly string EventHubName = "event-hub1";
+        private EventHubProducerClient eventHubProducerClient;
+        public ChangeFeedProcessor(EventHubProducerClient _eventHubProducerClient)

```

Comment:
This is a really unusual format.   A common C# convention is that class-level members are prefixed with an underscore to denote their scope.    If we're going to use prefixes, then we should be doing something similar - I've never seen parameters decorated like this.

Alternatively, we could make the member  `private readonly EventHubProducerClient EventHubProducerClient;` to disambiguate.   Either is fine, as would keeping the same private member name and disambiguating with `this`.

Reply:
@jsquire - Updated.